bib/internal/buildconfig/config.go

```diff
@@ -101,6 +101,15 @@ func loadConfig(path string) (*externalBlueprint.Blueprint, error) {
 	}
 }
 
+func LoadConfig(path string) (*imagesBlueprint.Blueprint, error) {
+	externalBp, err := loadConfig(path)
```

Comment:
(nitpick) I would prefer if we would follow the idiomatic go style here, i.e.
```go
    externalBp, err := loadConfig(path)
    if err != nil {
         return nil, err
    }
    bp := externalBlueprint.Convert(*externalBp)
    return &bp, nil
}
```
I know its the same but when reading (go) code, I just stumble on anything that does not follow the usual "foo, err := ...; if err != nil {" pattern.